lib/split/alternative.rb

```diff
@@ -166,7 +166,7 @@ def validate!
     end
 
     def reset
-      Split.redis.hmset key, "participant_count", 0, "completed_count", 0, "recorded_info", nil
+      Split.redis.hmset key, "participant_count", 0, "completed_count", 0, "recorded_info", nil.to_s
```

Comment:
```suggestion
      Split.redis.hmset key, "participant_count", 0, "completed_count", 0, "recorded_info", ""
```

On the previous redis nil would become an empty string.